### PR TITLE
Vault - Getting Started Demo/Test

### PR DESCRIPTION
**Pull request by j-pye**
This PR is for a demo / test deployment of Vault that can be used to confirm Vault is working in any of the namespaces attached to a project set. Further examples will eventually be placed her to go with the Service Definition and docs being added to the `openshift-wiki`

Explanations of the annotations used in the example will be fully documented in the other repository and a link will be added through another PR at a later time.

### CODE REVIEW

File: vault/getting-started-demo/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+  **Clean Up!!** - Let's not waste log space
+
+  `oc delete -f vault-$LICENSE_PLATE-$ENV`
+
+  Assuming your logs look similar to those shown above your project set is configured and setup for integrating Vault with your Applications! :tada:
+
+## Support
+
+For information on Vault and for getting Vault support from the community check out `devops-vault` on RocketChat!
```
Comment (sbathgate) [inline vault/getting-started-demo/README.md]: Strictly a nitpick, might not hurt to include a link to the devops-vault channel?
```suggestion
For information on Vault and for getting Vault support from the community check out [#devops-vault](https://chat.developer.gov.bc.ca/channel/devops-vault) on RocketChat!
```

**Review by sbathgate — APPROVED** [review]
LGTM, strictly a nitpick / thought for consideration below.

File: vault/getting-started-demo/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+  > **Note:** It is recommended to start with the tools
+
+  **Verify the Pod is up with `2/2` containers `READY`**
+
+  ```bash
+  oc get pods
+  NAME                         READY   STATUS    RESTARTS   AGE
+  vault-test-7c6cd9f45f-n4dnt   2/2    Running   0          1d
+  ```
```
Comment (StevenBarre) [inline vault/getting-started-demo/README.md]:
```suggestion
  ```console
  $ oc get pods
  NAME                         READY   STATUS    RESTARTS   AGE
  vault-test-7c6cd9f45f-n4dnt   2/2    Running   0          1d
  ```
```

Use console formatting when including output.

File: vault/getting-started-demo/README.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+  **Verify the Pod Logs are outputing `world`**
+
+  ```bash
+  oc logs deployment/vault-test -c vault
```
Comment (StevenBarre) [inline vault/getting-started-demo/README.md]:
```suggestion
  ```console
  $ oc logs deployment/vault-test -c vault
```

File: vault/getting-started-demo/README.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  world
+  ```
+
+  **View the Injected Secrets Template** (Optional)
+
+  Grab the pod name from `oc get pods`
+
+  ```bash
+  oc rsh -c vault vault-test-7c6cd9f45f-n4dnt 
+  $ cat /vault/secrets/helloworld
```
Comment (StevenBarre) [inline vault/getting-started-demo/README.md]:
```suggestion
  ```console
  $ oc rsh -c vault vault-test-7c6cd9f45f-n4dnt 
  $ cat /vault/secrets/helloworld
```

**Review by StevenBarre — APPROVED** [review]
Just some markdown formatting suggestions.

**Comment by j-pye** [comment]
TIL `console` is a codeblock type. Thanks!!

**Pull request merged**